packages/core/integration-tests/test/svg-react.js

```diff
@@ -19,4 +19,24 @@ describe('svg-react', function () {
     assert(file.includes('const SvgIcon ='));
     assert(file.includes('_react.createElement("svg"'));
   });
+  it('should support transforming SVGs to typescript react components', async function () {
+    let b = await bundle(
+      path.join(__dirname, '/integration/svg-react-typescript/react.ts'),
+      {
+        defaultConfig: path.join(
+          __dirname,
+          'integration/custom-configs/.parcelrc-svg-react',
+        ),
+        defaultTargetOptions: {
+          typescript: true,
+        },
```

Comment:
This option doesn't exist
```suggestion
```

Reply:
For some reason I thought those options were being passed to the plug-in, but that doesn’t make sense I realise